lib/Onnxifi/Flags.cpp

```diff
@@ -329,34 +329,34 @@ DEFINE_validator(glow_nnpi_num_parallel_chunks,
                  });
 #endif /* GLOW_WITH_NNPI */
 
-DEFINE_uint32(glow_interpreter_memory, 0,
+DEFINE_int32(glow_interpreter_memory, 0,
```

Comment:
Using this for compatibility issue. Some old gflag library doesn't supprt `DEFINE_uint32` and since the unit here is KiB, we the int32 range should be enough.